<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qdrm</string>
  </property>
  <property name="accessibleName">
   <string>Main window</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGeneral">
       <attribute name="title">
        <string>Genral Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="dMRIDLabel">
            <property name="text">
             <string>DMR ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="dmrID"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="introLine1Label">
            <property name="text">
             <string>Intro Line 1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="introLine1"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="introLine2Label">
            <property name="text">
             <string>Intro Line 2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="introLine2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabContacts">
       <attribute name="title">
        <string>Contacts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
            <item>
             <widget class="QTableView" name="contactsView">
              <property name="accessibleName">
               <string>contactsView</string>
              </property>
              <property name="accessibleDescription">
               <string>Lists all contacts.</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="contactUp">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="contactDown">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="addContact">
              <property name="text">
               <string>Add Contact</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remContact">
              <property name="text">
               <string>Delete Contact</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRXGroups">
       <attribute name="title">
        <string>RX Groups</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
            <item>
             <widget class="QListView" name="groupListView"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="rxGroupUp">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rxGroupDown">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="addRXGroup">
              <property name="text">
               <string>Add RX Group</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remRXGroup">
              <property name="text">
               <string>Delete RX Group</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabChannels">
       <attribute name="title">
        <string>Channels</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
            <item>
             <widget class="QTableView" name="channelView"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QPushButton" name="channelUp">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="channelDown">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="addAnalogChannel">
              <property name="text">
               <string>Add Analog Channel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addDigitalChannel">
              <property name="text">
               <string>Add Digital Channel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remChannel">
              <property name="text">
               <string>Delete Channel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabZones">
       <attribute name="title">
        <string>Zones</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QListView" name="zoneView"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QPushButton" name="zoneUp">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="zoneDown">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="addZone">
              <property name="text">
               <string>Add Zone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remZone">
              <property name="text">
               <string>Delete Zone</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabScanLists">
       <attribute name="title">
        <string>Scan Lists</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,0">
            <item>
             <widget class="QListView" name="scanListView"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QPushButton" name="scanListUp">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="scanListDown">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="addScanList">
              <property name="text">
               <string>Add Scan List</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remScanList">
              <property name="text">
               <string>Delete Scan List</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewCodeplug"/>
    <addaction name="actionCodeplugWizzard"/>
    <addaction name="actionOpenCodeplug"/>
    <addaction name="actionSaveCodeplug"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionDetectDevice"/>
    <addaction name="actionVerifyCodeplug"/>
    <addaction name="separator"/>
    <addaction name="actionDownload"/>
    <addaction name="actionUpload"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevice"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="accessibleName">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewCodeplug"/>
   <addaction name="actionCodeplugWizzard"/>
   <addaction name="actionOpenCodeplug"/>
   <addaction name="actionSaveCodeplug"/>
   <addaction name="separator"/>
   <addaction name="actionDetectDevice"/>
   <addaction name="actionVerifyCodeplug"/>
   <addaction name="actionDownload"/>
   <addaction name="actionUpload"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionNewCodeplug">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Creates a new Codeplug.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCodeplugWizzard">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/dialog.png</normaloff>:/icons/dialog.png</iconset>
   </property>
   <property name="text">
    <string>New with wizzard ...</string>
   </property>
   <property name="toolTip">
    <string>Creates a new codeplug with a wizzard.</string>
   </property>
  </action>
  <action name="actionOpenCodeplug">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open ...</string>
   </property>
   <property name="toolTip">
    <string>Imports a codeplug from CSV files.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveCodeplug">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save ...</string>
   </property>
   <property name="toolTip">
    <string>Saves the codeplug in a CSV file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quits the application.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDetectDevice">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>Detect</string>
   </property>
   <property name="toolTip">
    <string>Detect connected radios.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionVerifyCodeplug">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
   <property name="toolTip">
    <string>Verifies the codeplug with the connected radio.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/download.png</normaloff>:/icons/download.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>Downloads a codeplug from the connected device.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/upload.png</normaloff>:/icons/upload.png</iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
   <property name="toolTip">
    <string>Uploads the codeplug to the connected device.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About qdrm</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Opens the handbook.</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
